tests/runtest.py

```diff
@@ -10,8 +10,8 @@
 
 class TestBase:
     supported_lang = {
-        'C':   { 'cc': 'gcc', 'flags': 'CFLAGS',   'ext': '.c' },
-        'C++': { 'cc': 'g++', 'flags': 'CXXFLAGS', 'ext': '.cpp' },
+        'C':   { 'cmd1': 'gcc', 'cmd2': 'clang', 'flags': 'CFLAGS',   'ext': '.c' },
+        'C++': { 'cmd1': 'g++', 'cmd2': 'clang++', 'flags': 'CXXFLAGS', 'ext': '.cpp' },
```

Comment:
I doesn't look like a good idea to keep two different compilers in the dictionary. We can directly access them like `supported_lang['C']['cc']` and `supported_lang['C++']['cc']`.